project/AppDependencies.scala

```diff
@@ -3,24 +3,30 @@ import sbt._
 
 object AppDependencies {
 
+  private lazy val silencerVersion      = "1.7.9"
+  private lazy val hmrcMongoPlayVersion = "0.71.0"
+
+  //TODO migrate from jodaTime to JavaTime
```

Comment:
we can just change some imports and use that as we did the same in other projects as well, just run AT and make sure all good, why not?